### DISCUSSION TITLE
Placing Start

### DISCUSSION THREAD

**Discussion by sambiagz** [sk-zk/TruckLib#5] [Q&A] [answered]
Hi, is there a way to place the start using code ?
When I generate a map using the RealRoad code, I can't "place the start vehicle" on the road cause its either too high or too low below the road.

Thanks

**Comment by sk-zk** [comment]
The start location is encoded in a weird format not seen anywhere else in the map format, so I've yet to figure out how it works.

**Comment by sk-zk** [comment, marked as answer]
OK, I've just had another look. I don't fully remember what the problem was with the start position fields, but whatever it was, SCS must've changed it at some point because it's just global coordinates now (plus a weird extra value which can seemingly be ignored). This means that [you can now](https://github.com/sk-zk/TruckLib/commit/3a741002e6a1942285719a5d7b9e68f6ae14538f) set the start position with `map.StartPosition` and `map.StartRotation`.

**Reply by sambiagz** [reply]
Thanks works great.
